test/check-kubernetes

```diff
@@ -205,4 +205,61 @@ class TestConnection(KubernetesCase):
         output = m.execute("curl -k -sS https://localhost:6443/api 2>&1 || true")
         self.assertIn("Unauthorized", output)
 
+
+@unittest.skipIf("rhel-7" == os.environ.get("TEST_OS", ""), "Skipping check-nulecule on rhel-7.")
+class TestNulecule(KubernetesCase):
+    def setUp(self):
+        MachineCase.setUp(self)
+        m = self.machine
+
+        self.fix_apiserver_config()
+        m.execute("systemctl start etcd kube-apiserver kube-controller-manager kube-scheduler docker kube-proxy kubelet")
+
+        self.wait_api_server()
+
+    def testDeployDialog(self):
+        b = self.browser
+        m = self.machine
+        b.wait_timeout(120)
+
+        self.login_and_go("/kubernetes/cluster", host=None)
+        b.wait_present("#node-list")
+        b.wait_in_text("#node-list", "127.0.0.1")
+
+        # 1)check atomic version
+        output = m.execute("atomic -v")
+        self.assertTrue(output>=1.1)
```

Comment:
This will return true no matter what value ```atomic -v``` returns, because output is a string. Try it in python:

```
$ python -c 'print "1.0" > 1.1'
True
```